### PR TITLE
#164798011 Setup Coveralls code coverage service

### PR DESCRIPTION
**Pull request by walsamlee**
__What does this PR do?__
Integrates coveralls code coverage service

__Description of Task to be completed?__
* Setup Coveralls service

__How should this be manually tested?__
NA

__What are the relevant pivotal tracker stories?__
[#164798011](https://www.pivotaltracker.com/n/projects/2318856/stories/164798011)

__Screenshots__
Screenshot 2019-03-25 at 5 31 28 PM

### CODE REVIEW

**Comment by rotimi-babalola** [comment]
There are merge conflicts in this PR @walsamlee

**Comment by rotimi-babalola** [comment]
There are merge conflicts in this PR @walsamlee also please don't force push. Fix the merge conflicts in this PR in another commit

**Comment by walsamlee** [comment]
@rotimi-babalola conflict has been resolved

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      "require-jsdoc": ["error", {
+          "require": {
+              "FunctionDeclaration": true,
+              "MethodDefinition": true,
+              "ClassDeclaration": true
+          }
+      }]
+    }
+}
+
```
Comment (rotimi-babalola) [inline .eslintrc.json:34]: Why am I seeing this file again in this PR? This file is already on `develop` isn't it? 🤔

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (rotimi-babalola) [inline .gitignore]: You need to properly resolve merge conflicts in this file

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+# next.js build output
+.next
+=======
+node_modules
+>>>>>>> 3c5cf88... Chore(setup): add coveralls code coverage service
```
Comment (rotimi-babalola) [inline .gitignore]: Same as ☝🏾

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+=======
+node_modules
+>>>>>>> 3c5cf881d63e5d73b7718df14c79d578bce86d35
```
Comment (rotimi-babalola) [inline .gitignore]: Same as ☝🏾

**Review by rotimi-babalola — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (walsamlee) [reply .eslintrc.json:34]: i'm not sure i worked directly on this file, maybe it was when i rebased that it was altered.

**Pull request closed without merging**